config/routes.rb

```diff
@@ -8,7 +9,8 @@
     root controller: DashboardManifest::ROOT_DASHBOARD, action: :index
   end
 
-  devise_for :users
+  devise_for :users,
+    controllers: {omniauth_callbacks: 'users/omniauth_callbacks'}
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.